docs/developers/dapp-developer/native-bindings.md

```diff
@@ -40,13 +40,17 @@ pragma solidity >0.7.0 < 0.9.0;
 */
 
 contract Storage {
+	string public version;
+	uint256 public value;
 
-	uint256 value;
+	constructor(string memory _version)  {
```

Comment:
Why do you want to change value to version?

Reply:
Yes please revert